Need to explicitly call .as_matrix() here to get a vnl_matrix<double>.
author: Franky Backeljauw (Universiteit Antwerpen)
--- Examples/WarpVTKPolyDataMultiTransform.cxx.orig	2025-06-10 20:33:01.000000000 +0200
+++ Examples/WarpVTKPolyDataMultiTransform.cxx	2025-08-16 22:25:58.292843942 +0200
@@ -254,7 +254,7 @@
     point.CastFrom( It.Value() );
   */
   vnl_matrix_fixed<double, 4, 4> ijk2ras, vtk2ras, lps2ras;
-  vtk2ras = ConstructVTKtoNiftiTransform(field_output->GetDirection().GetVnlMatrix(),
+  vtk2ras = ConstructVTKtoNiftiTransform(field_output->GetDirection().GetVnlMatrix().as_matrix(),
                                          field_output->GetOrigin().GetVnlVector(),
                                          field_output->GetSpacing().GetVnlVector());
   ijk2ras.set_identity();
@@ -264,11 +264,11 @@
   lps2ras(1, 1) = -1;
 
   // Set up the transforms
-  ijk2ras = ConstructNiftiSform(field_output->GetDirection().GetVnlMatrix(),
+  ijk2ras = ConstructNiftiSform(field_output->GetDirection().GetVnlMatrix().as_matrix(),
                                 field_output->GetOrigin().GetVnlVector(),
                                 field_output->GetSpacing().GetVnlVector());
 
-  vtk2ras = ConstructVTKtoNiftiTransform(field_output->GetDirection().GetVnlMatrix(),
+  vtk2ras = ConstructVTKtoNiftiTransform(field_output->GetDirection().GetVnlMatrix().as_matrix(),
                                          field_output->GetOrigin().GetVnlVector(),
                                          field_output->GetSpacing().GetVnlVector());
 
